### DISCUSSION TITLE
Displays the number where the beatmap is being played relative to the total filtered beatmap.

### DISCUSSION THREAD

**Discussion by huyenden** [ppy/osu#24224] [Ideas] [answered]
The story is that I (or the player) after SS a beatmap and shutdown Osu!Lazer. Next time when I reopen Lazer, I want the next SS beatmap in the difficulty sequence, I don't know where the SS beatmap is on the slider. So I think if I remember the number (or percentage) of the beatmaps completed and shutdown after reopening, it would be easier to find them again. For example the image below:

![Untitled](https://github.com/ppy/osu/assets/128424762/b07393ee-ec60-4a7d-bd5b-db5134a26558)

Or can this be shown and hidden after dragging.
![Untitled1](https://github.com/ppy/osu/assets/128424762/19b568fa-8e9a-4c4f-a642-6bb145f90954)

**Comment by peppy** [comment, marked as answer]
You could sort by "last played" if you're looking for recently played beatmaps.
